packages/react-aria-components/docs/Tree.mdx

```diff
@@ -322,6 +833,108 @@ TreeItem also exposes a `--tree-item-level` CSS custom property, which you can u
 }
 ```
 
+### TreeItemContent
```

Comment:
TreeItem is missing selector for focus visible within?

Reply:
updating this one, ran into an issue, the data-* are not available on the TreeItemContent element, only on TreeItem. However, I can't realistically override every selector as it includes properties from other interfaces we inherit from. For example, isHovered and isPressed. I don't want them as data-* but I do still want their description and they are in the renderProps function.